readme.md

```diff
@@ -42,6 +45,70 @@ print(LaunchAtLogin.isEnabled)
 //=> true
 ```
 
+#### As singleton:
+
+```swift
+import LaunchAtLogin
+
+print(LaunchAtLogin.shared.isEnabled)
+//=> false
+
+LaunchAtLogin.shared.isEnabled = true
+
+print(LaunchAtLogin.shared.isEnabled)
+//=> true
+```
+
+#### Storyboards:
+
+Bind control to  `LaunchAtLogin.isEnabled` ViewController's property:
+
+```
+import Cocoa
+import LaunchAtLogin
+
+class ViewController: NSViewController {
+    @objc dynamic var launchAtLogin = LaunchAtLogin.shared
+}
+```
+![storyboard_binding](.github/storyboard_binding.png)
+
+#### SwiftUI:
+
+Use `LaunchAtLogin.shared` as binding with `@ObservedObject`:
+
+```
+import SwiftUI
+import LaunchAtLogin
+
+struct ContentView: View {
+    @ObservedObject private var launchAtLogin = LaunchAtLogin.shared
+
+    var body: some View {
+        Toggle(isOn: $launchAtLogin.isEnabled) {
+            Text("Launch at login")
+        }
+        .toggleStyle(CheckboxToggleStyle())
+    }
+}
+```
+#### Combine:
+
+Just subscribe to `LaunchAtLogin.publisher`:
+
+```
+import Combine
+import LaunchAtLogin
+
+private var isLaunchAtLoginEnabled = LaunchAtLogin.isEnabled
+private var cancellable = Set<AnyCancellable>()
+
+LaunchAtLogin
+    .publisher
+    .assign(to: \.isLaunchAtLoginEnabled, on: self)
```

Comment:
This is invalid as there's no `self` here. I would move it into a class to make the example more realistic.